ui-v2/app/services/repository/proxy.js

```diff
@@ -22,7 +22,11 @@ export default RepositoryService.extend({
   findInstanceBySlug: function(id, slug, dc, configuration) {
     return this.findAllBySlug(slug, dc, configuration).then(function(items) {
       if (get(items, 'length') > 0) {
-        const instance = items.findBy('ServiceProxyDestination', id);
+        let instance = items.findBy('ServiceProxyDestination', id);
+        if (instance) {
+          return instance;
+        }
+        instance = items.findBy('ServiceProxyDestination', slug);
```

Comment:
This field is now deprecated. We should use `ServiceProxy.DestinationServiceName` instead.

Reply:
Thanks Paul! Amended!